code/modules/surgery/advanced/bioware/nerve_grounding.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/disk/surgery/nerve_grounding
+	desc = "The disk provides instructions on how to reroute the nervous system to ground electric shocks."
+	surgeries = list(/datum/surgery/advanced/bioware/nerve_grounding)
+
+/datum/surgery/advanced/bioware/nerve_grounding
+	name = "nerve grounding"
+	steps = list(/datum/surgery_step/incise,
+				/datum/surgery_step/retract_skin,
+				/datum/surgery_step/clamp_bleeders,
+				/datum/surgery_step/incise,
+				/datum/surgery_step/incise,
+				/datum/surgery_step/ground_nerves,
+				/datum/surgery_step/close)
+	possible_locs = list("chest")
+	bioware_target = "nerves"
+
+/datum/surgery_step/ground_nerves
+	name = "ground nerves"
+	accept_hand = TRUE
+	time = 155
+
+/datum/surgery_step/ground_nerves/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] starts splicing together [target]'s nerves.", "<span class='notice'>You start splicing together [target]'s nerves.</span>")
+
+/datum/surgery_step/ground_nerves/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] successfully splices [target]'s nervous system!", "<span class='notice'>You successfully splice [target]'s nervous system!</span>")
+	new /datum/bioware/threaded_veins(target)
```

Comment:
all of these surgeries that spawn a new bioware on the target use the threaded veins path incorrectly.